src/JToric.jl

```diff
@@ -13,8 +13,15 @@ Initializing function for 'JToric'.
 # initialization
 function __init__()
     # load necessary gap packages
-    GAP.Globals.LoadPackage( GapObj( "JConvex" ) )
-    GAP.Globals.LoadPackage( GapObj( "ToricVarieties" ) )
+    if ( ! GAP.Packages.load( "NConvex", "2021.04-24", install = false ) )
+             warn("Could not load desired version of GAP package NConvex. JToric may not be fully functional.")
+    end
+    if ( ! GAP.Packages.load( "JConvex", "2021.08.12", install = false ) )
```

Comment:
Why "install = false" in two cases, and "true" in the third?

Reply:
The development version of `NConvex` that I am using is not distributed in GAP. Same for `JConvex`. The only distributed package, which could be installed is `ToricVarieties`.

Reply:
@fingolfin Is this acceptable? I.e. can this be merged? It reduces the coverage slightly, as the above warnings are never triggered during the tests, but I would think this is ok.